WBooks/Screens/BookDetail/BookDetailView.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  BookDetailView.swift
+//  WBooks
+//
+//  Created by Federico Agustin Diaz on 05/08/2019.
+//  Copyright © 2019 Wolox. All rights reserved.
+//
+
+import UIKit
+import WolmoCore
+
+protocol DetailBookDelegate: class {
+    func rentBook()
+}
+
+class BookDetailView: UITableViewCell, NibLoadable {
+    
+    @IBOutlet weak var bookTitle: UILabel!
```

Comment:
In the screencast, this title is not showing the whole text. Make it take 2 lines